front_end/src/components/booking/BookingBreakdownTable.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { withStyles } from "@material-ui/core/styles";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import TableFooter from "@material-ui/core/TableFooter";
+import Paper from "@material-ui/core/Paper";
+
+const CustomTableCell = withStyles(theme => ({
+  head: {
+    backgroundColor: theme.palette.common.black,
+    color: theme.palette.common.white,
+    fontSize: 14
+  },
+  body: {
+    fontSize: 14
+  },
+  footer: {
+    backgroundColor: "#84ffff",
```

Comment:
can you use our secondary color? #a0cdfd

Reply:
changes made